interface-definitions/interfaces_pppoe.xml.in

```diff
@@ -20,7 +20,19 @@
           #include <include/interface/authentication.xml.i>
           #include <include/interface/dial-on-demand.xml.i>
           #include <include/interface/no-default-route.xml.i>
-          #include <include/interface/default-route-distance.xml.i>
+          <leafNode name="default-route-distance">
+            <properties>
+              <help>Distance for installed default route</help>
+              <valueHelp>
+                <format>u32:1-255</format>
+                <description>Distance for the default route from DHCP server</description>
```

Comment:
PPPoE uses PPP options for auto-configuration, and it's a distinct mechanism from DHCP, so this help string is kinda misleading. Maybe say just "the server".

```suggestion
                <description>Distance for the default route received from the server</description>
```

Reply:
done

Reply:
Please put the article back to make the sentence grammatical. Note that updating the code by hand instead of committing my suggestion made the process longer for us both. ;)

Reply:
fixed :) Also i can't apply your suggestion directly because changes were made in another file and `#included` instead of copy past as was in the first version :)